src/lib/holocene/loading.svelte

```diff
@@ -1,19 +1,28 @@
 <script lang="ts">
   import { translate } from '$lib/i18n/translate';
-  import Logo from '$lib/vendor/Temporal_Logo_Animation.gif';
+  import mp4Logo from '$lib/vendor/Temporal_Logo_Animation.mp4';
+  import webmLogo from '$lib/vendor/Temporal_Logo_Animation.webm';
 
   export let title = translate('common.loading');
 </script>
 
-<div class="my-12 flex flex-col items-center justify-start {$$props.class}">
-  <img
-    src={Logo}
-    style="margin-top: -40px;"
-    alt=""
-    width="200px"
-    height="200px"
-  />
-  <h2 class="text-xl font-medium" style="margin-top: -40px;">
+<div
+  class="my-12 flex flex-col items-center justify-start gap-4 {$$props.class}"
+>
+  <video
+    autoplay
+    loop
+    muted
```

Comment:
> Autoplay blocking is not applied to [<video>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) elements when the source media does not have an audio track, or if the audio track is muted.

@rossedfort I had assumed [this](https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide) ⬆️  meant we could get around autoplay blocking.

But after further testing it definitely does not work in certain browsers with `autoplay` blocked (e.g. Firefox). I've gone ahead and [added a check](https://github.com/temporalio/ui/pull/2021/commits/8a1cc2789937a3d5ba8bc311186d57c35e3fb4ec) to see if it is allowed and a fallback to a logo ⬇️ if it is not.

![Screenshot 2024-04-25 at 12 28 40 PM](https://github.com/temporalio/ui/assets/15069288/63d63aac-87ff-467c-a5aa-7006de46dd35)